scripts/github_api

```diff
@@ -93,13 +109,30 @@ class PullRequest:
         self._set_attributes(
             request('POST', self._url, data=json.dumps(payload_body)))
 
+    def get(self, pr_number):
+        """
+        Gets a GitHub pull-request
```

Comment:
AV> Just trying to make it obvious that this is a specific thing and not any "request to pull". There's code already merged with the dash so there's not much I can do now.

Sorry, but the dash don't make that obvious to me *at all*, I think readability is better without it. Of course you can do, you just need to checkout master, create a branch, replace all occurrences of 'pull-request' with 'pull request', submit a patch and rebase this PR on top of it. It takes a few minutes to do it. Up to you, I wont fight over it.